docs/examples/Generic Branch Example.ipynb

```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 3,
```

Comment:
can you please restart kernel, clear outputs and rerun?

I'd expect this number to be `0`

Reply:
that's odd, which interpreter are you using? jupyter notebook or raw ipykernel (e.g. using the VS Code plugin for notebooks)